support-workers/src/main/scala/com/gu/emailservices/EmailFields.scala

```diff
@@ -40,17 +40,3 @@ case class EmailFields(
   }
 
 }
-
-case class AllProductsEmailFields(
-  billingPeriod: BillingPeriod,
-  user: User,
-  currency: Currency,
-  sfContactId: SfContactId,
-  directDebitMandateId: Option[String],
-)
-
-case class SubscriptionEmailFields(
-  allProductsEmailFields: AllProductsEmailFields,
-  subscriptionNumber: String,
-  promotion: Option[Promotion] = None
-)
```

Comment:
these are no longer needed as the new type is a far neater way of gathering the appropriate values together.